server/middleware/file.js

```diff
@@ -1,6 +1,7 @@
 const multer = require('multer');
 const multerS3 = require('multer-s3');
 const uuid = require('uuid/v4');
+const querystring = require('querystring');
```

Comment:
Does this need to be in `package.json`? If it's in core, comment it as such with `// core`.